packages/comms/src/clienttools/eclcc.ts

```diff
@@ -145,6 +120,91 @@ export class EclccErrors {
     }
 }
 
+export class EclccErrors extends Errors {
+
+    constructor(stdErr: string, checked: string[]) {
+        super(checked);
+        if (stdErr && stdErr.length) {
+            for (const errLine of stdErr.split(os.EOL)) {
+                let match = /([a-z,A-Z]:\\(?:[-\w\.\d]+\\)*(?:[-\w\.\d]+)?|(?:\/[\w\.\-]+)+)\((\d*),(\d*)\): ?(error|warning|info) C(\d*): ?(.*)/.exec(errLine);
```

Comment:
It looks like this is trying to match a file path? Maybe `match` should be renamed to `isFilePath` for clarity?

Reply:
Only part of the "match" is a file path, its also matching error code and message.